### PR TITLE
Feature/documentation readme update

### PR DESCRIPTION
**Pull request by tylercherpak**
### Description of the Change

Moving documentation into the readme of this repo. This servers to reduce the barrier of entry to get involved with the project or add to the documentation. 

This will also make it easier to keep the documentation up to date and require changes to docs when introducing changes.

### Benefits
Easier to manage and contribute to.

### Possible Drawbacks
The readme is getting big.

### Verification Process
Read the docs.

### Checklist:

- [x] I have read the [**CONTRIBUTING**](/CONTRIBUTING.md) document.
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [ ] All new and existing tests passed.

### Changelog Entry

Updating all the docs and consolidating them into the readme of the repo instead of its own repo.

### CODE REVIEW

**Comment by TheLastCicada** [comment]
What happens to the old docs site?  Can we redirect it back to this?

File: README.md [hunk 1]
```diff
@@ -6,11 +6,263 @@
...
+
+### Elasticsearch
+
+If you have enabled Elasticsearch for a particular environment, you can send requests from the host machine to the
+Elasticsearch server by prefixing the url path with `/__elasticsearch/`. For example, if you wanted to hit the
+`/_all/_search/` endpoint of Elasticsearch, the URL would look like: `http://<hostname>/__elasticsearch/_all/_search`
+
+### WP Snapshots
+
+See the section on [using WP Snapshots](../wp-snapshots)
```
Comment (joesnellpdx) [inline README.md:143]: @tylercherpak 

This needs to be updated or add the section from [here](https://10up.github.io/wp-local-docker-docs/wp-snapshots/).

**Comment by tylercherpak** [comment]
> What happens to the old docs site? Can we redirect it back to this?

I am not sure what we can do in this instance since its github pages. how would one redirect from github pages? Or we can update that repo just to say: "This is deprecated, click here to see updated docs." and then link to this readme. 

cc: @dustinrue

**Comment by jeffpaul** [comment]
@tylercherpak there might be something we could do to push an update to the docs repo that effectively generates a GitHub Pages site that just redirects to the new docs site.  I'll do some research to see if that's possible and if not updating the landing page to note the new link would be Plan B.

**Comment by jeffpaul** [comment]
https://dev.to/steveblue/setup-a-redirect-on-github-pages-1ok7 & https://gist.github.com/domenic/1f286d415559b56d725bee51a62c24a7 could help with redirect.  We'll want to keep the old docs repo active for a bit for folks to get that redirect, but eventually can archive that repo.

**Pull request merged**